docs/data/charts/overview/overview.md

```diff
@@ -7,34 +7,29 @@ packageName: '@mui/x-charts'
 
 # MUI X Charts
 
-<p class="description">A fast and extendable library of react chart components for data visualization.</p>
+<p class="description">A collection of React chart components for data visualization.</p>
```

Comment:
This might impact SEO.
Are we sure that removing the `extendable` word here isn't for the worse? 🤔 
cc @alexfauquette

Reply:
The correct word choice here would be "extensible", and I'd be surprised if it were part of a long-tail keyword. We already use "fast and extensible" to describe the Data Grid throughout the docs as well.